k8s/upgrades/0.8.2-0.9.0/jiva/jiva_upgrade_runtask.yaml

```diff
@@ -329,6 +316,82 @@ spec:
 ---
 apiVersion: openebs.io/v1alpha1
 kind: RunTask
+metadata:
+  name: upgrade-jiva-volume-0.8.2-0.9.0-pre-check-ctrl-pod-phase
+  namespace: default
+spec:
+  meta: |
+    id: listCtrlPods
+    runNamespace: {{ .UpgradeItem.namespace }}
+    apiVersion: v1
+    kind: Pod
+    action: list
+    options: |-
+      labelSelector: openebs.io/persistent-volume={{ .UpgradeItem.name }},openebs.io/controller=jiva-controller
+  post: |
+    {{- $CustomJsonPath := printf "{.items[?(@.status.phase=='Running')].metadata.name}" -}}
+
+    {{- jsonpath .JsonResult $CustomJsonPath | trim | saveAs "listCtrlPods.podName" .TaskResult | noop -}}
+    {{- .TaskResult.listCtrlPods.podName | notFoundErr "controller pod is not in running phase" | saveIf "listCtrlPods.notFoundErr" .TaskResult | noop -}}
```

Comment:
or change the message to `No controller pods found with running phase`

Reply:
Updated the error message